static/src/javascripts/lib/closest.spec.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import closest from './closest';
+
+const closestProto = Element.prototype.closest;
+let nestedChild;
+
+beforeAll(() => {
+    // Remove the closest method from Element so we test the polyfill
+    delete Element.prototype.closest;
+
+    if (document.body) {
+        document.body.innerHTML = `
+        <div class="greatgrandparent">
+            <div class="greatgrandparent findthisone">
+                <div class="grandparent">
+                    <div class="parent">
+                        <div class="child" id="child">child content</div>
+                    </div>
+                </div>
+            </div>
+        </div>`;
+    }
+
+    nestedChild = document.querySelector('.child');
+});
+
+afterAll(() => {
+    // Flow moans that I'm assigning something to a prototype
+    // $FlowFixMe
```

Comment:
Oh wait no, this is the opposite problem! I guess Flow considers `closest` to be covariant (why? Is it defined this way somewhere?). And [you can't write to covariant properties](https://flowtype.org/blog/2016/10/04/Property-Variance.html#property-variance)